chat-core/src/DatabaseConnection.ts

```diff
@@ -15,6 +15,7 @@ export type DatabaseConnection = {
     @param force - Force close, emitting no events
    */
   close(force?: boolean): Promise<void>;
+  drop(): Promise<void>;
```

Comment:
I had this as `_debugDropDatabase()`. This makes it extra obvious that it should only be used on tests.

```suggestion
  _debugDropDatabase(): Promise<void>;
```